src/models/generator_models/avr_models.jl

```diff
@@ -100,18 +100,17 @@ function mdl_avr_ode!(
     Se_Vf = Ae * exp(Be * abs(Vf)) #16.13
     V_ref = V0_ref + Vs
 
-    #Set anti-windup for Vr1. #TODO in callbacks
-    #if Vr1 > Vr_max
-    #    Vr1 = Vr_max
-    #elseif Vr1 < Vr_min
-    #    Vr1 = Vr_min
-    #end
+    # Compute block derivatives
+    _, dVm_dt = low_pass(V_th, Vm, 1.0, Tr)
+    y_hp, dVr2_dt = high_pass(Vf, Vr2, Kf, Tf)
+    _, dVr1_dt = low_pass(V_ref - Vm - y_hp, Vr1, Ka, Ta)
+    _, dVf_dt = low_pass_modified(Vr1, Vf, 1.0, Ke + Se_Vf, Te)
 
     #Compute 4 States AVR ODE:
-    output_ode[local_ix[1]] = -(1.0 / Te) * (Vf * (Ke + Se_Vf) - Vr1) #16.12c
-    output_ode[local_ix[2]] = (1.0 / Ta) * (Ka * (V_ref - Vm - Vr2 - (Kf / Tf) * Vf) - Vr1) #16.12a
-    output_ode[local_ix[3]] = -(1.0 / Tf) * ((Kf / Tf) * Vf + Vr2) #16.12b
-    output_ode[local_ix[4]] = (1.0 / Tr) * (V_th - Vm) #16.11
+    output_ode[local_ix[1]] = dVf_dt #16.12c
```

Comment:
We should do this instead where needed. 
```suggestion
    output_ode[local_ix[1]] = low_pass_modified(Vr1, Vf, 1.0, Ke + Se_Vf, Te)[2] #16.12c
```